pkgs/apps/sharc/21.nix

```diff
@@ -91,6 +93,10 @@ in stdenv.mkDerivation {
     done
   '';
 
+  setupHooks = [
```

Comment:
Does that work with `nix-shell -p sharc`?

Reply:
Actually yes! (Very much to my surprise) :smile: (So with `nix-shell -p qchem.sharcV21`)

Reply:
Nice